__tests__/src/components/TicketsTable.test.tsx

```diff
@@ -0,0 +1,131 @@
+import '@testing-library/jest-dom';
+import {
+  render, screen, renderHook, waitFor,
+} from '@testing-library/react';
+import TicketsTable from '@components/TicketsTable';
+import type { ServiceRequestSummary } from '@types';
+
+import { recentCases } from '@hooks/__mocks__/useRecentTickets';
+import useRecentTickets from '@hooks/useRecentTickets';
+import ClientService from '@services/ClientService';
+
+jest.mock('@services/ClientService');
+const mockedClientService = jest.mocked(ClientService);
+
+beforeAll(() => {
+  // Setup mock ClientService
+  mockedClientService.getServiceRequestSummary
+    .mockImplementation(async () => recentCases.map((ticket) => ({
+      ...ticket,
+      client: ticket.client_id,
+      pet: ticket.pet_id,
+      team_member: ticket.team_member_id,
+    })));
+});
+
+export interface TicketsTableProps {
+  items: ServiceRequestSummary[]
+}
+
+describe('TicketTable Headers', () => {
+  it('Checks Headers Presence', async () => {
+    //* Act
+    const { result } = renderHook(useRecentTickets);
+    //* Assert
+    render(<TicketsTable items={result.current} />);
+
+    /*
+    [{"client": "456def",
+    "client_id": "456def",
+    "created_at": "",
+    "description": "Cleaning service",
+    "id": "147xyz",
+    "log_id": "",
+    "pet": "789ghi",
+    "pet_id": "789ghi",
+    "request_source_id":
+    "phone-id",
+    "service_category_id":
+    "pet_fostering-id",
+    "team_member": "j456",
+    "team_member_id": "j456"}]
+
+    */
```

Comment:
Nit: It's best to not leave in commented out code. Not a big deal

Reply:
I can remove that, while I was writing test, wanted to see what I was getting.